### PR TITLE
Printstylesheets updated

### PR DESCRIPTION
**Pull request by KJLammers**
Printstylesheets now compatible with printing of attributes, scale and
flow lay-out

### CODE REVIEW

File: viewer/src/main/webapp/WEB-INF/xsl/print/A3_Landscape.xsl [hunk 1]
```diff
@@ -1,196 +1,284 @@
...
+	<xsl:variable name="global-scale" select="''"/>
+	<!-- omrekening van pixels naar mm -->
+	<xsl:variable name="ppm" select="'2.8'"/>
+	<!-- See legend.xsl (does not currently affect size of other elements!) -->
+	<xsl:variable name="legend-width-cm" select="5.6"/>
+	<!-- See legend.xsl ('none', 'before', 'right') -->
+	<xsl:variable name="legend-labels-pos" select="'before'"/>
+	<xsl:variable name="legend-scale-images-same-ratio" select="true()"/>
+	<!-- formatter -->
+	<xsl:decimal-format name="MyFormat" decimal-separator="." grouping-separator="," infinity="INFINITY" minus-sign="-" NaN="Not a Number" percent="%" per-mille="m" zero-digit="0" digit="#" pattern-separator=";"/>
```
Comment (mprins) [inline viewer/src/main/webapp/WEB-INF/xsl/print/A3_Landscape.xsl]: can you restore this really long line to multiple lines

File: viewer/src/main/webapp/WEB-INF/xsl/print/A3_Landscape.xsl [hunk 2]
```diff
@@ -1,196 +1,284 @@
...
+						<fo:block xsl:use-attribute-sets="header-font">
+							<xsl:value-of select="ancestor::info[1]/@componentname"/>
+						</fo:block>
+						<xsl:call-template name="table-related"/>
+					</xsl:for-each>
+					<xsl:if test="count(//extra/info[@classname='attributes']) = 0">
+						<fo:block xsl:use-attribute-sets="default-font">
+							<xsl:text></xsl:text>
+						</fo:block>
+					</xsl:if>
```
Comment (mprins) [inline viewer/src/main/webapp/WEB-INF/xsl/print/A3_Landscape.xsl]: basically this is the only new stuff, apart from some pixel shifting, right?

File: viewer/src/main/webapp/WEB-INF/xsl/print/A3_Landscape.xsl [hunk 3]
```diff
@@ -1,196 +1,284 @@
...
+			<xsl:value-of select="imageUrl"/>
+			<xsl:text>&amp;width=</xsl:text>
+			<xsl:value-of select="$map-width-px-corrected"/>
+			<xsl:text>&amp;height=</xsl:text>
+			<xsl:value-of select="$map-height-px-corrected"/>
+			<xsl:text>&amp;bbox=</xsl:text>
+			<xsl:value-of select="$bbox-corrected"/>
+		</xsl:variable>
+		<fo:block>
+			<fo:external-graphic src="{$map}" content-height="scale-to-fit" content-width="scale-to-fit" scaling="uniform" width="{$map-width-px}" height="{$map-height-px}" xsl:use-attribute-sets="simple-border"/>
```
Comment (mprins) [inline viewer/src/main/webapp/WEB-INF/xsl/print/A3_Landscape.xsl]: please break this line

**Review by mprins — COMMENTED** [review]
Not happy with reformatting and whitespace changes, makes it basically impossible to review

**Review by mtoonen — APPROVED** [review]

**Pull request merged**